syncd/FlexCounter.cpp

```diff
@@ -2122,6 +2224,19 @@ void FlexCounter::addCounter(
 
             setSwitchDebugCounterList(vid, rid, switchCounterIds);
         }
+        else if (objectType == SAI_OBJECT_TYPE_MACSEC_SA && field == MACSEC_SA_ATTR_ID_LIST)
+        {
+            std::vector<sai_macsec_sa_attr_t> MACsecSAIds;
+
+            for (const auto &str : idStrings)
+            {
+                int32_t attr;
+                sai_deserialize_enum(str, &sai_metadata_enum_sai_macsec_sa_attr_t, attr);
```

Comment:
ok, i actually look at rest, so every other attribute have dedicated sai_deserialize_xxx_attr function for this specific purpose, so i think you can also add this dedicated function to saiserialize.cpp as you had previously, just to follow convention, sorry for this, my mistake

Reply:
Done, please review it.